contributor/Testing_Request.md

```diff
@@ -0,0 +1,11 @@
+# Testing Request
+The OpenLab provides two kinds of resources: VM(Virtual Machine) and PM(Physical Machine) for testing. Currently we only utilize virtual machines to support automating jobs for SDKs and tools integration test with OpenLab CI. On the other hand you can request PMs for your other testing.
+
+1. Submit a test request to OpenLab, fill out some information like what kind of machines (VMs or PMs) and how many do you expect to use, what testing are you planning to implement or need assistance implementing and so on, for more details see [test request](https://github.com/theopenlab/openlab/issues/new?template=integration_test_request.md) 
+2. The governance team will confirm the request to check whether it is acceptable, if yes then will add it to [TODO list](https://github.com/orgs/theopenlab/projects/1#column-1860008)
+3. The test request in TODO list can be assigned, if you wish to work on it please pick it up, move the item to [IN PROGRESS list](https://github.com/orgs/theopenlab/projects/1#column-1860011), edit the item to add the development plan
```

Comment:
1. s/if you wish/if you or another OpenLab members wish/
2. s/it/the item/
3. s/edit the item to add the development plan/edit it to add the development plan into comments/